app/serializers/recipe/index_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class Recipe::IndexSerializer < ActiveModel::Serializer
+  attributes :id, :title, :instructions, :created_at, :updated_at
```

Comment:
On the client we would like to receive a humanized date format.
You can either choose a new name like:
```
  def published
    object.created_at.to_formatted_s(:long)
  end
  ```
  or use the same "created_at" name:
  ```
  def created_at
    object.created_at.to_formatted_s(:long)
  end
  ```